end-to-end_sales-forecasting/sales_forecasting-end-to-end.ipynb

```diff
@@ -0,0 +1,2701 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.            ## 2022 update: this probably is no longer necessary, either because timestamps are included as index in data files, and/or that we will no longer require manual set of begin &amp; end timestamps in near future, for monitoring ingestion
</pre><p>Can we just remove the timestamp as extra data if its no longer necessary (and this comment)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/truera-examples/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Done</p>